cfme/networks/provider/__init__.py

```diff
@@ -25,8 +25,7 @@
     OneProviderNetworkRouterView,
     OneProviderSecurityGroupView,
     OneProviderSubnetView,
-    NetworkProviderEditView
-)
+    NetworkProviderEditView)
```

Comment:
Is this change really needed here?

Reply:
No, sorry, this was overwritten by a rebase merge.